### ISSUE TITLE
Language name exported at beginning of source code block

### ISSUE THREAD

**Issue by dhermanson** [ofosos/ox-epub#14]
Hello. I'm using GNU Emacs 25.1.1 on macOS. Thank you for this amazing package.

When I do an export to .epub using this package, source code blocks seem to put the name of the language as the first part of the exported source code, which doesn't seem desirable to me. For example:

```ruby
Rubyclass Person
  def initialize name, age
    @name = name
    @age = age
  end
end
```

```python
Pythonclass Person:
    def __init__(self, name, age):
        self.name = name
        self.age = age
```

```emacs-lisp
Emacs Lisp(setq value "example")
```

Thanks!

**Comment by ofosos** [comment]
Didn't test that, looking into it on the weekend. Thanks for the report.

**Comment by ofosos** [comment]
Sorry for the late answer, but I can't currently reproduce that. I'm using:

```
#+begin_src python
def foo(x):
  if x>0:
    return x+1

  else:
    return x-1

return foo(5)
#+end_src

```

**Comment by ofosos** [comment]
Wait... this is indeed undesirable. This is fixed in v0.2.3.
